projects/v2/src/app/static/config.ts

```diff
@@ -243,6 +261,10 @@ export const SHEET_OPTIONS = [
     title: 'Lung DRAFT',
     sheet: 'lung',
   },
+  {
+    title: 'Lung v1.0',
+    sheet: 'lungv1',
```

Comment:
lungv1 => lung_v1.0 (I'm imagining the scenario when we start using versions and that's how I see it being transformed)